src/nbsap/forms.py

```diff
@@ -483,6 +534,39 @@ def save(self):
         return indicator
 
 
+class NationalIndicatorEditForm(NationalIndicatorForm, ChoicesMixin):
+    code = forms.CharField(max_length=16, validators=[validate_eu_target_code],
```

Comment:
How about we remove the code field completely? It was anyway added to the NationalIndicatorForm.